policies/design-process.md

```diff
@@ -0,0 +1,141 @@
+Policy for Creating Designs
+===========================
+
+From time to time the OTC may require that certain problems should create a
```

Comment:
_...certain problems should have a design document..._

The problems aren't creating anything :)